composer.json

```diff
@@ -101,12 +102,14 @@
         "serialssolutions/summon": "1.3.1",
         "slm/locale": "1.2.0",
         "steverhoades/oauth2-openid-connect-server": "3.0.1",
+        "stripe/stripe-php": "v17.5.0",
         "swagger-api/swagger-ui": "5.21.0",
         "symfony/console": "6.4.21",
         "symfony/mailer": "6.4.21",
         "symfony/rate-limiter": "^6.4",
         "symfony/var-dumper": "6.4.21",
         "symfony/yaml": "6.4.21",
+        "tecnickcom/tcpdf": "6.10.0",
```

Comment:
As you may already know, this library seems to be in the process of getting deprecated in favor of https://github.com/tecnickcom/tc-lib-pdf -- since the new one is not stable yet, I'm not sure there's much we can do about it right now, but something we'll need to keep an eye on.

Reply:
I did not have any success with tc-lib-pdf, but decided to refactor the whole thing to use mPdf so that the contents can be created with a pretty normal phtml template. Much easier to manage and customize now, I believe.